src/execution_plan/execution_plan.c

```diff
@@ -1041,11 +1026,10 @@ static void _ExecutionPlan_FinalizeProfiling(OpBase *root) {
 	root->stats->profileExecTime *= 1000;   // Milliseconds.
 }
 
-ResultSet *ExecutionPlan_Profile(ExecutionPlan *plan) {
+void ExecutionPlan_Profile(ExecutionPlan *plan) {
 	_ExecutionPlan_InitProfiling(plan->root);
-	ResultSet *rs = ExecutionPlan_Execute(plan);
+	ExecutionPlan_Execute(plan);
 	_ExecutionPlan_FinalizeProfiling(plan->root);
-	return rs;
 }
```

Comment:
Similar to `ExecutionPlan_Execute` `ExecutionPlan_Profile` should return a result-set.
the caller is expected to free it.